Dshop/apps/products_catalogue/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+class CategoryBaseModel(models.Model):
+    name = models.CharField(max_length=200, unique=True)
+    slug = models.SlugField(max_length=200, unique=True)
+    is_active = models.BooleanField(default=True)
+    sort = models.IntegerField(default=99)
```

Comment:
please change default to max value for integer fied. Also it should be "PositiveSmallIntegerField" no? Max val 32767

Reply:
Actually we decided to drop sort.

Reply:
Done :)